apps/coordinator/src/components/Wallet/TransactionsTab/FeeBumping/context/feeBumpHooks.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from "react";
+import { useFeeBumpContext } from "./FeeBumpContext";
+import { FeeBumpStrategy } from "@caravan/fees";
+
+// Transaction data hooks
+export function useFeeBumpTransaction() {
+  const { state } = useFeeBumpContext();
+  return state.transaction;
+}
```

Comment:
I don't know if all of these are necessary. whenever you need a piece of the context in a component you can just do:

```typescript
const { txhex, status } = useUseFeeBumpContext()
```
(is it necessary to nest it even under `state`?

Reply:
Ah I see, it is for when you're using `useReducer` so that's fine. Still feels like, especially with Typescript and intellisense, this code is kind of unnecessary. If you want to do a `useGetFeeBumpState()` and then destructure from the state, that'd be fine, but all of these functions don't really give you much that you don't already get from typescript type hints.

Reply:
Looks like you even have that at the bottom of the file! you just aren't using it.